airflow/providers/google/cloud/hooks/dataflow.py

```diff
@@ -331,6 +361,12 @@ def cancel(self) -> None:
                     )
                 )
             batch.execute()
+            if self._cancel_timeout and isinstance(self._cancel_timeout, int):
```

Comment:
What other type of field do you envision here? The method signature uses ʻint`. Did you want to support `Optional[Int]` here?

Reply:
Yes. I updated type annotations to `Optional[Int]`. Thank you for pointing this out.